src/logicLayer/GameMap.java

```diff
@@ -2,11 +2,22 @@
 
 public class GameMap
 {
-	private Tile[][] tile;
+	private Tile[][] tiles;
 	private int turnCount;
 	
+	public GameMap(Tile[][] tile)
+	{
+		tiles = tile;
+		turnCount=1;
+	}
+	
 	public void increaseTurn()
 	{
 		turnCount++;
 	}
+	
+	public Tile[][] getTiles()
+	{
+		return tiles;
+	}

```

Comment:
Please add a method called getTileAt(Position position) that returns the tile at a given position